1907-07-01.xml

```diff
@@ -0,0 +1,1180 @@
+                                <cell>Tons 2,300</cell>
+                                <cell>Ocean Prince</cell>
+                                <cell>Tons 2,450</cell>
+                            </row>
+                        </table>
+                        <p>Good Accommodation for Passengers. -- C. J. GRACE &amp; Co., Alexandria, Agents.</p>
+                        <p>Sailings every 10 days from Manchester and Liverpool and fortnightly from Antwerp
+                            and London to Alexandria and Syrian Coast. The dates are approximate</p>
+                        <table rows="4" cols="8">
+                            <row>
+                                <cell>Egyptian Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Manchester</placeName></cell>
+                                <cell>about <date when="1907-03-26">March 26</date></cell>
+                                <cell>Belgian Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Manchester</placeName></cell>
+                                <cell><date when="1907-04-08">April 8</date></cell>
+                            </row>
+                            <row>
+                                <cell>Scottish Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Antwerp</placeName> &amp;
+                                    <placeName>London</placeName></cell>
+                                <cell><date when="1907-03-30">March 30</date></cell>
+                                <cell>Persian Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Antwerp</placeName> &amp;
+                                    <placeName>London</placeName></cell>
+                                <cell><date when="1907-04-09">April 9</date></cell>
+                            </row>
+                            <row>
+                                <cell>Creole Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Antwerp</placeName> &amp;
+                                    <placeName>London</placeName></cell>
+                                <cell><date when="1907-03-31">March 31</date></cell>
+                                <cell>Kaffir Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Manchester</placeName></cell>
+                                <cell><date when="1907-04-16">April 16</date></cell>
+                            </row>
+                        </table>
+                        <p>The S.S. Trojan Prince is now loading for <placeName>Manchester</placeName>.</p>
+                    </div>
+                    <!-- Summer Trips to Cypress -->
+                    </div>
```

Comment:
this `</div>` tag closes the advertising section--it should come just before the end of the page.